api/openapi-spec/swagger.json

```diff
@@ -10673,21 +10673,15 @@
     "properties": {
      "limits": {
       "description": "Limits describes the maximum amount of compute resources allowed. Valid resource keys are \"memory\" and \"cpu\".",
-      "type": "object",
-      "additionalProperties": {
-       "$ref": "#/definitions/k8s.io.apimachinery.pkg.api.resource.Quantity"
-      }
+      "type": "object"
```

Comment:
What I don't understand is, why this never resolved itself to `string`, since that is what the helper method on the quantity object indicates.

Reply:
`Quantity` will resolve itself to `string`. I thing the type `object` and `additionalProperties` comes with the fact that `limits`/`requests` are `maps`.